src/minerva/services/note_operations.py

```
@@ -500,3 +508,242 @@ def _should_auto_update_index(self) -> bool:
             and self.config.auto_index_enabled
             and self.config.vector_db_path is not None
         )
+
+    @log_performance(threshold_ms=1000)
+    @validate_inputs(validate_filename)
+    @handle_file_operations()
+    def merge_notes(
```

Pylint (3.3.7)

[refactor] Too many arguments (10/5)

(R0913)